gradle.properties

```diff
@@ -1,7 +1,7 @@
 # Main properties
 group=com.truelayer
 archivesBaseName=truelayer-java
-version=10.3.1
+version=10.4.0
```

Comment:
Minor version bump as we're exposing the commercial VRP capability via a new predefined scope constant